gcd/config/index.yaml

```diff
@@ -1,6 +1,25 @@
 indexes:
+
+  # Common index for all the applications.
+
   - kind: spine.base.Event
     ancestor: no
     properties:
     - name: type
     - name: created
+
+  # Indexes for the Aggregates.
+  #
+  # Each custom Aggregate type must have such indexes in order to be retrieved properly.
+  #
+  # The Datastore kind represents the Protobuf type name of the Aggregate state. Both the property
+  # names and the index directions should be kept unchanged.
+
+  - kind: spine.test.aggregate.Project
```

Comment:
If this is a sample of how an index entry should look like, why isn't it a part of a comment block?

I would suggest re-naming the kind to be ending with `MyAggregate`, and the package being `myapp.example` or something like that. We need to highlight the fact that it doesn't belong to Spine.